data/json/items/migration.json

```diff
@@ -20,6 +20,12 @@
     "replace": "battery_car"
   },
   {
+    "id": "gold_small",
+    "type": "MIGRATION",
+    "replace": "gold",
+    "charges": 1
```

Comment:
Yes. Given those items currently receive little to zero use I didn't consider this to be a significant regression versus adding more complex migration code.